core/model/snapshot.go

```diff
@@ -105,6 +110,19 @@ func GetSnapshotList(from, size int, taskId string) (int, []Snapshot, error) {
 	return result.Total, snapshots, err
 }
 
+func GetSnapshotAllList(taskId string) (int, []Snapshot, error) {
```

Comment:
this method is duplicated with GetSnapshotList, 
and should always limit record size before fetch.